src/main/webapp/js/community-page-loader.js

```diff
@@ -1,33 +1,57 @@
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+const h = React.createElement;
+const root = document.getElementById("list");
 
-/** Fetches users and adds them to the page. */
-function fetchUserList(){
-const url = '/user-list';
-fetch(url).then((response) => {
-  return response.json();
-}).then((users) => {
-  const list = document.getElementById('list');
-  list.innerHTML = '';
+class Community extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.fetchUserList = this.fetchUserList.bind(this);
+  }
+  /** Adds list of community users to community.html as a list element */
+  render() {
+    const { Users } = this.state;
```

Comment:
Components are capital camel case, and variables and functions are lower camel case.